src/java/com/rapleaf/jack/Column.java

```diff
@@ -0,0 +1,72 @@
+package com.rapleaf.jack;
+
+public class Column {
+  private static String DEFAULT_ID_FIELD = "id";
+
+  protected String table;
+  protected final Enum field;
+  protected final Class type;
+
+  protected Column(String table, Enum field, Class type) {
+    this.table = table;
+    this.field = field;
+    this.type = type;
+  }
+
+  protected Column(Column that) {
+    this.table = that.table;
+    this.field = that.field;
+    this.type = that.type;
+  }
+
+  public static Column fromKey(String table) {
+    return new Column(table, null, Long.class);
```

Comment:
If we are going to have `Integer` primary keys in the future, the method may be renamed as `fromLongId`, so that later on we can add a `fromIntId` counterpart.